assets/public/js/main.js

```diff
@@ -6,3 +8,93 @@ function toggleCartDropdown() {
 }
 
 cartBtn.onclick = toggleCartDropdown;
+
+/* Navigation menus */
```

Comment:
Do not leave comments in your code. Instead try to structure your code in such a way that you don't need them

Reply:
Changed.